### DISCUSSION TITLE
kernel header directory for manjaro

### DISCUSSION THREAD

**Discussion by payrim** [evilsocket/opensnitch#1091] [General]
**Describe the bug**
```
<built-in>:1:10: fatal error: '/lib/modules/6.6.10-1-MANJARO/source/include/linux/kconfig.h' file not found
#include "/lib/modules/6.6.10-1-MANJARO/source/include/linux/kconfig.h"
```

Include the following information:
 - OpenSnitch version 1.6.4
 - OS: manjaro
 - Version [e.g. Buster, 10.3, 20.04]
 - Kernel version: 6.6.10-1-MANJARO

**To Reproduce**
manjaro's kernel headers are somewhere else. it would be nice so this would be integrated

**Comment by gustavo-iniguez-goya** [comment]
hey @payrim ,

Sorry, I'm not really sure what this issue is about. Did you try to compile the ebpf modules from the sources? or did you try to install opensnitch-ebpf-modules package?

It seems that a similar problem was reported here: https://aur.archlinux.org/packages/opensnitch-ebpf-module
